Screen-Sharing/Screen-Sharing/TBScreenCapture.m

```diff
@@ -136,8 +136,8 @@ - (int32_t)stopCapture
     _capturing = NO;
     
     dispatch_sync(_queue, ^{
-        if (_timer) {
-            dispatch_source_cancel(_timer);
+        if (self->_timer) {
```

Comment:
`self->_timer` ?

I think it should be `self.timer`, since we are in a `.m` file instead of a `.mm`

Reply:
I think we usually use dot notation instead, but it's fine, we can leave as it is